### ISSUE TITLE
Making note taking web app responsive

### ISSUE THREAD

**Issue by savanpatel** [standardnotes/web#40]
Currently, the app is not responsive. Hence it is also not  mobile friendly. I would suggest to making it responsive.
I can pick up that task if required.

**Comment by mobitar** [comment]
The web app is really meant to be a desktop web app. For mobile, you'd use the mobile apps. How do you envision a responsive layout looking?

**Comment by savanpatel** [comment]
Well users nowdays prefer to use webapps directly through browser instead of installing it as app as it saves memory too. For example, as they do while surfing facebook.

Wouldn't it be nice by supporting responsiveness!

**Comment by mobitar** [comment]
Good point. But in this case the mobile apps would be more secure in general. But again how would you envision a responsive layout looking? I've tried to do some by simply rearranging the columns but it didn't feel right. Do you have other ideas?

**Comment by savanpatel** [comment]
Yes, I do. I am working on it. I will post images and updates here and lets see if it feels right.

**Comment by mobitar** [comment]
Cool

**Comment by levlaz** [comment]
Would love to see this as well. I use the web app to take notes, but I like to have the note pad take up about 1/4 of my screen. Would be nice if it folded properly to allow me to take notes while looking at other stuff on the other 3/4 of my screen.

**Comment by DDanielH** [comment]
How is responsive going? 
Still some design Problems? I would make it like the App with a Sidebar and such things.

**Comment by savanpatel** [comment]
I am facing issues in deciding the proper layout for mobile view. Like deciding where each of sections will fall. Sidebar menu like options sounds good. May be I can use help or we can discuss design together.

**Comment by mobitar** [comment]
I have a "responsive" branch going, and it's almost done. But I had to switch focus. There are a couple issues with it that can be easily fixed. If anyone wants to take a look this would be a good place to start: https://github.com/standardnotes/web/tree/responsive
